spec/ibandit/iban_spec.rb

```diff
@@ -1178,11 +1180,12 @@
           specify { expect(iban.errors).to include(bank_code: 'is ongeldig') }
         end
 
-        context 'when the bank code is not required' do
+        context 'when the branch code is also invalid' do
```

Comment:
Would prefer to see this as a context at the same level as `with an invalid bank code` and `with an invalid account number` - the three should be independent.